packages/opencensus-core/src/trace/model/tracer.ts

```diff
@@ -162,34 +159,30 @@ export class CoreTracer implements types.Tracer {
     });
   }
 
+  /** Notifies listeners of the span start. */
   onStartSpan(root: types.RootSpan): void {
-    if (this.active) {
-      if (!root) {
-        return this.logger.debug('cannot start trace - no active trace found');
-      }
-      if (this.currentRootSpan !== root) {
-        this.logger.debug(
-            'currentRootSpan != root on notifyStart. Need more investigation.');
-      }
-      this.notifyStartSpan(root);
+    if (!this.active) return;
+    if (!root) {
+      return this.logger.debug('cannot start trace - no active trace found');
```

Comment:
What would you think about separating the `return` from the logger statement? Though it compiles like this, it looks unexpected to me as if it were returning a non-undefined value from the logger statement.

Reply:
Good catch, done.